packages/quantic/force-app/main/default/lwc/quanticRadioButtonsGroup/__tests__/quanticRadioButtonsGroup.test.js

```diff
@@ -0,0 +1,289 @@
+  legend: 'legend.radio-buttons-group__legend',
+  radioInputs: 'input[type="radio"]',
+  radioLabels: 'label.radio-buttons-group__label',
+  radioIcons: 'lightning-icon',
+  radioContainers: '.slds-button.slds-radio_button.radio-button',
+  radioTooltips: 'c-quantic-tooltip',
+};
+
+const defaultOptionsList = [
+  {
+    value: 'value_option1',
+    tooltip: 'tooltip_option1',
+    label: 'label_option1',
+  },
+  {
+    value: 'value_option2',
+    tooltip: 'tooltip_option2',
+    label: 'label_option2',
+  },
+  {
+    value: 'value_option3',
+    tooltip: 'tooltip_option3',
+    label: 'label_option3',
+  },
+];
+
+const exampleOptions = {
+  legend: 'example-legend',
+  options: defaultOptionsList,
+};
+
+function createTestComponent(options = exampleOptions) {
+  const element = createElement('c-quantic-radio-buttons-group', {
+    is: QuanticRadioButtonsGroup,
+  });
+
+  for (const [key, value] of Object.entries(options)) {
+    element[key] = value;
+  }
+
+  document.body.appendChild(element);
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function flushPromises() {
+  // eslint-disable-next-line @lwc/lwc/no-async-operation
+  return new Promise((resolve) => setTimeout(resolve, 0));
+}
+
+describe('c-quantic-radio-buttons-group', () => {
+  function cleanup() {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+    jest.clearAllMocks();
+  }
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  describe('default rendering options', () => {
+    it('should render correctly with default options, without labels, without icons, without tooltips', async () => {
+      const minimalOptions = [
+        {value: 'value_option1'},
+        {value: 'value_option2'},
+        {value: 'value_option3'},
+      ];
+      // @ts-ignore
+      const element = createTestComponent({
+        ...exampleOptions,
+        // @ts-ignore
+        options: minimalOptions,
+      });
+      await flushPromises();
+
+      const legendElement = element.shadowRoot.querySelector(selectors.legend);
+      const radioInputs = element.shadowRoot.querySelectorAll(
+        selectors.radioInputs
+      );
+      const radioLabels = element.shadowRoot.querySelectorAll(
+        selectors.radioLabels
+      );
+      const icons = element.shadowRoot.querySelectorAll(selectors.radioIcons);
+
+      expect(legendElement).not.toBeNull();
+      expect(legendElement.textContent).toBe(exampleOptions.legend);
+      expect(radioInputs).toHaveLength(minimalOptions.length);
+      expect(radioLabels).toHaveLength(0);
+      expect(icons).toHaveLength(0);
+      minimalOptions.forEach((option, index) => {
+        expect(radioInputs[index].value).toBe(option.value);
+      });
+    });
+
+    it('should render correctly with default options, without icons', async () => {
+      // @ts-ignore
```

Comment:
Why do we need a ts-ignore here?

Reply:
dunno was probably left during my testing